components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -473,6 +478,8 @@ void RewardsServiceImpl::OnWalletProperties(ledger::Result result,
 void RewardsServiceImpl::OnGrant(ledger::Result result,
                                  const ledger::Grant& grant) {
   TriggerOnGrant(result, grant);
+  bat_notifications_service_->AddNotification(
```

Comment:
This seemed to be the only place to add a notification like this so far, but let me know.

Reply:
yes this is the only place for now